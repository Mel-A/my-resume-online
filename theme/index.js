const theme = {
  colors: {
    alabaster: '#FAFAFA',
    astral: '#327DA0',
    aquamarine: '#A1DAD7',
    azure: '#0080FF',
    cobalt: '#003DB3',
    gray: '#333333',
    white: '#FFFFFF'
  },
  fonts: {
    small0: '10px',
    small8: '18px',
    medium0: '20px',
    medium4: '24px',
    big0: '30px'
  }
}

export default theme