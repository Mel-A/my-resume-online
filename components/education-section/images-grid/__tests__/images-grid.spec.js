import { shallow, mount } from 'enzyme'
import ImagesGrid from '..'
import { ThemeProvider } from "styled-components"
import theme from '../../../../theme'

jest.mock('react-i18next', () => ({
  useTranslation: () => ({t: key => key})
}))

describe('imagesGrid', () => {
  it('imagesGrid renders correctly', () => {
    const imagesGrid = shallow(<ImagesGrid />)
    expect(imagesGrid).toMatchSnapshot()
    
  })

  it('imagesGrid renders theme correctly', () => {
    const imagesGrid = mount(<ImagesGrid />, {
      wrappingComponent: ThemeProvider,
      wrappingComponentProps: {
        theme,
      },
    })
    expect(imagesGrid).toMatchSnapshot()
  })
})

