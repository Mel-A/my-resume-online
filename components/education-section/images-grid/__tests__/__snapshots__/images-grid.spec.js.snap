// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`imagesGrid imagesGrid renders correctly 1`] = `ShallowWrapper {}`;

exports[`imagesGrid imagesGrid renders theme correctly 1`] = `ReactWrapper {}`;
